src/foam/nanos/u2/navigation/SignUp.js

```diff
@@ -257,25 +154,21 @@ foam.CLASS({
   actions: [
     {
       name: 'login',
-      label: 'Get Started',
+      label: 'Create account',
```

Comment:
?? why change?

Reply:
Was trying to keep as consistent to pre-refactor Sign Up as possible. Can undo if needed.